app/views/medicaid/insurance_current_member/edit.html.erb

```diff
@@ -0,0 +1,34 @@
+<% content_for :header_title, "Health Coverage Needs" %>
+
+<div class="form-card">
+  <header class="form-card__header">
+    <div class="form-card__title">
+      Tell us who is currently enrolled in a health insurance plan.
+    </div>
+    <% if @step.members_not_needing_insurance.present? %>
+      <p class="text--help text--centered">
+        You've already indicated that <%=@step.members_not_needing_insurance.map(&:full_name).to_sentence %>
+        <%= @step.members_not_needing_insurance.count == 1 ? "is" : "are" %>
```

Comment:
could define a custom inflection and use `pluralize` here https://stackoverflow.com/questions/3517989/ruby-on-rails-how-do-you-explicitly-define-plural-names-and-singular-names-in-r

but maybe not worth it

another option:

- throw the whole string into a localization with a `one` and `other` option
- pass in `@step.members_not_needing_insurance.map(&:full_name).to_sentence` as a variable to that localization

Reply:
✅